design-at-cornell/src/explore-courses/title/FilterCheckbox.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { FilterCheckboxProps } from "../types/PropertyTypes";
+import { Checkbox, FilterCheckboxContainer } from "../ExploreCoursesStyles";
+import check from '../../static/images/check.svg'
+
+const FilterCheckbox = ({ label, checked, onClick }: FilterCheckboxProps) => (
```

Comment:
Your custom checkboxes work well! HTML also has `<input type="checkbox">` with some built-in functionalities that might decrease some of the functions and types that you need to maintain. But everything is well integrated so I don't think it's necessary to change.

Reply:
Thanks so much! I tried using the input element at first, but had a lot of difficulty styling them. From what I could find online, styling default checkboxes can be pretty challenging.